community/kernel-api/src/main/java/org/neo4j/internal/kernel/api/ExplicitIndexWrite.java

```diff
@@ -19,6 +19,12 @@
  */
 package org.neo4j.internal.kernel.api;
 
+import java.util.Map;
+
+import org.neo4j.internal.kernel.api.exceptions.KernelException;
+
 public interface ExplicitIndexWrite
```

Comment:
What's the reason for not having `nodeAddToExplicitIndex` here as well?

Reply:
it is coming to PR near you soon, stay tuned!